prime-router/assets/csv-download-site/login__inline.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="none" />
+
+    <title>DataHub - Login</title>
+    <script src="https://global.oktacdn.com/okta-signin-widget/5.2.1/js/okta-sign-in.min.js" type="text/javascript"></script>
+    <link href="https://global.oktacdn.com/okta-signin-widget/5.2.1/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+    <div id="okta-login-container"></div>
+</body>
+
+<script type="text/javascript" th:inline="javascript">
+    config = {
+        logo: '//logo.clearbit.com/cdc.gov',
+        language: 'en',
+        features: {
+            registration: false, // Enable self-service registration flow
+            rememberMe: false, // Setting to false will remove the checkbox to save username
+            router: true, // Leave this set to true for the API demo
+        },
+        el: "#okta-login-container",
+        baseUrl: 'https://'+ [[${OKTA_baseUrl}]],
+        clientId: [[${OKTA_clientId}]],
+        redirectUri: [[${OKTA_redirect}]],
+        authParams: {
+          issuer: "https://"+[[${OKTA_baseUrl}]]+"/oauth2/default"
+        }
+    };
+    
+    function createCookie(name,value,min) {
+        if (min) {
+            var date = new Date(); 
+            date.setTime(date.getTime()+(min*60*60*1000));   
+            var expires = ";expires="+date.toUTCString(); 
```

Comment:
Isn't `min*60*60*1000` an hours calculation not a minute? Assuming `min` means minutes.

Reply:
Oops - yes, changed to min * 60 * 1000